src/navigation/stripe_count.hpp

```diff
@@ -0,0 +1,142 @@
+/*
+ * Author:Justus Rudolph
+ * Organisation: HYPED
+ * Date: 07/03/2020
+ * Description: Stripe counter class for navigation.
+ *
+ *    Copyright 2019 HYPED
+ *    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ *    except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *    either express or implied. See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#ifndef NAVIGATION_STRIPE_COUNT_HPP_
+#define NAVIGATION_STRIPE_COUNT_HPP_
+
+#include <math.h>
```

Comment:
Can you try changing this to `<cmath>`? (`<math.h>` is the C version, whereas `<cmath>` is the C++ version of the same library.) The main difference is that the C version does not put stuff in the `std` namespace so it might cause naming conflicts. When you change it, I suppose it will break the calls to `abs()` and other functions in `stripe_count.cpp` when they're not preceded by `std::`. (Those should be made consistent anyway - either use `std::abs()` everywhere or add a `using` statement for it and use just `abs()`.)
```suggestion
#include <cmath>
```